### ISSUE TITLE
Layer-wise learning rate decay during fine-tuning

### ISSUE THREAD

**Issue by ziw-liu** [mehta-lab/VisCy#75]
BERT-style layer-wise LR decay following ConvNext v2 during fine-tuning.
